masonry/src/contexts.rs

```diff
@@ -856,6 +859,16 @@ impl LayoutCtx<'_> {
         self.get_child_state(child).layout_rect().size()
     }
 
+    /// Compute layout of a child widget.
+    ///
+    /// Container widgets must call this on every child as part of
+    /// their [`layout`] method.
+    ///
+    /// [`layout`]: Widget::layout
+    pub fn run_layout<W: Widget>(&mut self, child: &mut WidgetPod<W>, bc: &BoxConstraints) -> Size {
```

Comment:
We should ensure that this method is clearly pointed to in the docs for `Widget::layout`.

I'd also encourage moving this method higher up in this impl block, because it is among the most important methods on `LayoutCtx`